mod/01_rating/spec/set/type/metric_value_spec.rb

```diff
@@ -54,6 +54,31 @@
       value_card = Card["#{@metric_value.name}+value"]
       expect(value_card.content).to eq("I'm fine, I'm just not happy.")
     end
+
+    it 'creates metric value with source' do
+      url = 'http://www.google.com/?q=everybodylies'
+      source = Card::Set::Self::Source.find_duplicates(url).first.cardname.left
+      subcard = {
+        '+metric' => { 'content' => @metric.name },
+        '+company' => {
+          'content' => "[[#{@company.name}]]",
+          'type_id' => Card::PointerID
+        },
+        '+value' => {
+          'content' => "I'm fine, I'm just not happy.",
+          'type_id' => Card::PhraseID
+        },
+        '+year' => { 'content' => '2014', 'type_id' => Card::PointerID },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.